public/js/views/configpageView/genres.coffee

```diff
@@ -0,0 +1,84 @@
+$ = require "jquery"
+
+BaseView = require "../baseView.coffee"
+template = require "../../templates/configpageTemplate/genres.hbs"
+animtemplate = require "../../templates/configpageTemplate/animlist.hbs"
+
+
+Animations = require "../../helpers/animations.coffee"
+
+module.exports = BaseView.extend
+  el: "#genres"
+  template: template
+  initialize:(options) ->
+    @parent = options.parent
+    @animArr = Animations.getAll()
+  render: ->
+    $(@el).html @template({
+        animations: @animArr
+    })
+    return @
+  events: ->
+    "click .genre-btn" : "filterAnimations"
+    "click .anim-btn": "playAnimation" 
+
+  filterAnimations: (e) ->
+    if e.currentTarget.id == "gen-gifs"
+      $(".genre-btn").removeClass("active")
+      $("#gen-gifs").addClass("active")
+      $("#anim-list-group").html animtemplate({
+        gifs: @gifArr
+      })
+    else 
+      $("#gen-gifs").removeClass("active")
+      # TODO grab the value and filter animations accordingly
+      $("#anim-list-group").html animtemplate({
+        animations: @animArr
+      })
+
+  playAnimation: (e) ->
+    @clearSelection()
+    if @parent.getCurrentMappingCode()
+      @updateAnimation(e)
+    else
+      @animate(@getAnimId(e))
+
+  updateAnimation: (e) ->
+    anim = @getAnimId(e)
+    @animate(anim)
+    @makeSelection(anim)
+    @updateParent(anim)
+
+  getAnimId: (e) ->
+    ele = $(e.currentTarget)
+    animId = ele.val()
+    
+  animate: (anim) ->
+    return if anim == ""
+    @parent.play(anim)
+
+  makeSelection: (anim) ->
+    id = "anim-"+anim
+    $(document.getElementById(id)).addClass("active")
```

Comment:
Are you making sure that you're adding "#" before the ID in the jQuery selector? They also need to be in quotes.